BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -252,6 +252,10 @@ NormalBootPath (
     // It is a FV format
     DEBUG ((DEBUG_INFO, "FV Format Payload\n"));
     Status = LoadFvImage (Dst, Stage2Hob->PayloadActualLength, (VOID **)&PldEntry);
+    if ((FixedPcdGetBool (PcdPreOsCheckerEnabled))) {
+      KernelEntry = (UINT32) Dst;
+      DEBUG ((DEBUG_INFO, "KernelEntry 0x%08X\n", KernelEntry));
```

Comment:
The print should be FV base instead of "Kernel Entry".  
Maybe we should just rename KernelEntry to a more generic name, such as  PldBase ?